<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>Form</class>
 <widget class="QWidget" name="Form">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>803</width>
    <height>522</height>
   </rect>
  </property>
  <property name="font">
   <font>
    <family>Segoe UI</family>
   </font>
  </property>
  <property name="windowTitle">
   <string>Trim</string>
  </property>
  <property name="windowIcon">
   <iconset resource="../resources/resources.qrc">
    <normaloff>:/icons/icons/logo.svg</normaloff>:/icons/icons/logo.svg</iconset>
  </property>
  <property name="styleSheet">
   <string notr="true">QWidget {
	color: white;
	font-family: &quot;Segoe UI&quot;;
}</string>
  </property>
  <layout class="QVBoxLayout" name="verticalLayout">
   <property name="spacing">
    <number>0</number>
   </property>
   <property name="leftMargin">
    <number>0</number>
   </property>
   <property name="topMargin">
    <number>0</number>
   </property>
   <property name="rightMargin">
    <number>0</number>
   </property>
   <property name="bottomMargin">
    <number>0</number>
   </property>
   <item>
    <widget class="QFrame" name="base_frame">
     <property name="styleSheet">
      <string notr="true">QFrame#base_frame {
	background-color: #262626;
	/*border-radius: 10px;*/

}
</string>
     </property>
     <property name="frameShape">
      <enum>QFrame::StyledPanel</enum>
     </property>
     <property name="frameShadow">
      <enum>QFrame::Raised</enum>
     </property>
     <layout class="QVBoxLayout" name="verticalLayout_2">
      <property name="spacing">
       <number>0</number>
      </property>
      <property name="leftMargin">
       <number>0</number>
      </property>
      <property name="topMargin">
       <number>0</number>
      </property>
      <property name="rightMargin">
       <number>0</number>
      </property>
      <property name="bottomMargin">
       <number>0</number>
      </property>
      <item alignment="Qt::AlignTop">
       <widget class="QFrame" name="title_bar">
        <property name="sizePolicy">
         <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
          <horstretch>0</horstretch>
          <verstretch>0</verstretch>
         </sizepolicy>
        </property>
        <property name="minimumSize">
         <size>
          <width>0</width>
          <height>30</height>
         </size>
        </property>
        <property name="maximumSize">
         <size>
          <width>16777215</width>
          <height>30</height>
         </size>
        </property>
        <property name="styleSheet">
         <string notr="true">QFrame {
	background-color: #f77f00;
	/*border-top-left-radius: 10px;
	border-top-right-radius: 10px;
	border-bottom-right-radius: 0px;
	border-bottom-left-radius: 0px;*/
}</string>
        </property>
        <property name="frameShape">
         <enum>QFrame::NoFrame</enum>
        </property>
        <property name="frameShadow">
         <enum>QFrame::Raised</enum>
        </property>
        <layout class="QHBoxLayout" name="horizontalLayout_3">
         <property name="spacing">
          <number>0</number>
         </property>
         <property name="leftMargin">
          <number>0</number>
         </property>
         <property name="topMargin">
          <number>0</number>
         </property>
         <property name="rightMargin">
          <number>0</number>
         </property>
         <item>
          <widget class="QLabel" name="title">
           <property name="minimumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="maximumSize">
            <size>
             <width>50</width>
             <height>30</height>
            </size>
           </property>
           <property name="styleSheet">
            <string notr="true">QLabel {
	color: white;
	font-family: &quot;Segoe UI&quot;;
	font-weight: bold;

}</string>
           </property>
           <property name="text">
            <string>TRIM</string>
           </property>
           <property name="alignment">
            <set>Qt::AlignCenter</set>
           </property>
          </widget>
         </item>
         <item>
          <spacer name="spacer_title_bar">
           <property name="orientation">
            <enum>Qt::Horizontal</enum>
           </property>
           <property name="sizeHint" stdset="0">
            <size>
             <width>677</width>
             <height>18</height>
            </size>
           </property>
          </spacer>
         </item>
        </layout>
       </widget>
      </item>
      <item>
       <widget class="QFrame" name="main_frame">
        <layout class="QVBoxLayout" name="main_layout">
         <property name="spacing">
          <number>11</number>
         </property>
         <property name="leftMargin">
          <number>10</number>
         </property>
         <property name="topMargin">
          <number>10</number>
         </property>
         <property name="rightMargin">
          <number>10</number>
         </property>
         <property name="bottomMargin">
          <number>10</number>
         </property>
         <item>
          <widget class="QFrame" name="video_frame">
           <property name="sizePolicy">
            <sizepolicy hsizetype="Preferred" vsizetype="Expanding">
             <horstretch>0</horstretch>
             <verstretch>0</verstretch>
            </sizepolicy>
           </property>
           <property name="styleSheet">
            <string notr="true">QFrame {
	background-color: none;
	border: 1px solid #7f7f7f;
	border-radius: 0px;
}</string>
           </property>
           <layout class="QVBoxLayout" name="verticalLayout_3">
            <property name="spacing">
             <number>0</number>
            </property>
            <property name="leftMargin">
             <number>0</number>
            </property>
            <property name="topMargin">
             <number>0</number>
            </property>
            <property name="rightMargin">
             <number>0</number>
            </property>
            <property name="bottomMargin">
             <number>0</number>
            </property>
            <item>
             <widget class="QVideoWidget" name="video_widget" native="true">
              <property name="styleSheet">
               <string notr="true">QWidget {
	background-color: black;
}</string>
              </property>
             </widget>
            </item>
           </layout>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout">
           <property name="spacing">
            <number>5</number>
           </property>
           <property name="topMargin">
            <number>0</number>
           </property>
           <property name="rightMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="play_btn">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="font">
              <font>
               <family>Segoe UI</family>
               <pointsize>10</pointsize>
               <weight>75</weight>
               <bold>true</bold>
              </font>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: white;
	background-color: rgb(33, 213, 93);
	border-radius: 5px;
	font-weight: bold;
}

QPushButton:hover {
	border: 1px solid white;
}

QPushButton:pressed {
	background-color: rgb(27, 172, 75);
}

QPushButton:disabled {
	background-color: rgb(80, 80, 80);
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/icons/play_arrow_white_24dp.svg</normaloff>:/icons/icons/play_arrow_white_24dp.svg</iconset>
             </property>
             <property name="iconSize">
              <size>
               <width>16</width>
               <height>16</height>
              </size>
             </property>
             <property name="checkable">
              <bool>false</bool>
             </property>
             <property name="checked">
              <bool>false</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="set_ss_btn">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: white;
	background-color: none;
	border-radius: 5px;
	font-weight: bold;
}

QPushButton:hover {
	border: 1px solid #7f7f7f;
}

QPushButton:pressed {
	background-color: #444444;
}

QPushButton:disabled {
	background-color: rgb(80, 80, 80);
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/icons/align_horizontal_left_white_24dp.svg</normaloff>:/icons/icons/align_horizontal_left_white_24dp.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="set_to_btn">
             <property name="minimumSize">
              <size>
               <width>30</width>
               <height>30</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: white;
	background-color: none;
	border-radius: 5px;
	font-weight: bold;
}

QPushButton:hover {
	border: 1px solid #7f7f7f;
}

QPushButton:pressed {
	background-color: #444444;
}

QPushButton:disabled {
	background-color: rgb(80, 80, 80);
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/icons/align_horizontal_right_white_24dp.svg</normaloff>:/icons/icons/align_horizontal_right_white_24dp.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QSlider" name="slider">
             <property name="styleSheet">
              <string notr="true">QSlider::groove:horizontal {
	border: 1px solid #7f7f7f;
	background: rgb(24, 24, 24);
	height: 15px;
	border-radius: 7px;
}

QSlider::handle:horizontal {
	background: white;
	/* border: 1px solid #777; */
	width: 15px;
	border-radius: 7px;
}

/*QSlider::sub-page:horizontal {
	background: #f77f00;
	border: 1px solid #777;
	height: 10px;
	border-radius: 7px;
}</string>
             </property>
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
            </widget>
           </item>
           <item alignment="Qt::AlignVCenter">
            <widget class="QLabel" name="current_time">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Preferred" vsizetype="Preferred">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>60</width>
               <height>0</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: #7f7f7f;
}</string>
             </property>
             <property name="text">
              <string>00:00.000</string>
             </property>
             <property name="textFormat">
              <enum>Qt::PlainText</enum>
             </property>
             <property name="alignment">
              <set>Qt::AlignCenter</set>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_2">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="ss_minus_btn">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: white;
	background-color: none;
	border-radius: 5px;
	font-weight: bold;
}

QPushButton:hover {
	border: 1px solid #7f7f7f;
}

QPushButton:pressed {
	background-color: #444444;
}

QPushButton:disabled {
	background-color: rgb(80, 80, 80);
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/icons/minus_white_24dp.svg</normaloff>:/icons/icons/minus_white_24dp.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="ss_time">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	background-color: #333333;
	color: white;
	border: 1px solid #7f7f7f;
	border-radius: 5px;
	padding: 2px
}

QLineEdit:focus {
	border: 1px solid #f77f00;
}

QLineEdit:disabled {
	background-color: rgb(80, 80, 80);
	border-color: rgb(80, 80, 80);

}</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="ss_plus_btn">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: white;
	background-color: none;
	border-radius: 5px;
	font-weight: bold;
}

QPushButton:hover {
	border: 1px solid #7f7f7f;
}

QPushButton:pressed {
	background-color: #444444;
}

QPushButton:disabled {
	background-color: rgb(80, 80, 80);
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/icons/add_white_24dp.svg</normaloff>:/icons/icons/add_white_24dp.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QSpinBox" name="increment_spinbox">
             <property name="sizePolicy">
              <sizepolicy hsizetype="Expanding" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>0</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>60</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QSpinBox {
	background-color: #333333;
	color: white;
	border: 1px solid #7f7f7f;
	border-radius: 5px;
	padding: 2px;
	padding-left: 4px;

}

QSpinBox:hover {
	border: 1px solid #f77f00;
}

QSpinBox:focus {
	border: 1px solid #f77f00;
}

QSpinBox:disabled {
	background-color: #555555;
	color: #262626;
	border-color: rgb(80, 80, 80);
}

</string>
             </property>
             <property name="minimum">
              <number>10</number>
             </property>
             <property name="maximum">
              <number>10000</number>
             </property>
             <property name="singleStep">
              <number>10</number>
             </property>
             <property name="value">
              <number>100</number>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLabel" name="increment_label">
             <property name="styleSheet">
              <string notr="true">QLabel {
	color: #7f7f7f;
}</string>
             </property>
             <property name="text">
              <string>ms</string>
             </property>
             <property name="alignment">
              <set>Qt::AlignLeading|Qt::AlignLeft|Qt::AlignVCenter</set>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_2">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="to_minus_btn">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: white;
	background-color: none;
	border-radius: 5px;
	font-weight: bold;
}

QPushButton:hover {
	border: 1px solid #7f7f7f;
}

QPushButton:pressed {
	background-color: #444444;
}

QPushButton:disabled {
	background-color: rgb(80, 80, 80);
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/icons/minus_white_24dp.svg</normaloff>:/icons/icons/minus_white_24dp.svg</iconset>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QLineEdit" name="to_time">
             <property name="enabled">
              <bool>true</bool>
             </property>
             <property name="sizePolicy">
              <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
               <horstretch>0</horstretch>
               <verstretch>0</verstretch>
              </sizepolicy>
             </property>
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>0</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QLineEdit {
	background-color: #333333;
	color: white;
	border: 1px solid #7f7f7f;
	border-radius: 5px;
	padding: 2px
}

QLineEdit:focus {
	border: 1px solid #f77f00;
}

QLineEdit:disabled {
	background-color: rgb(80, 80, 80);
	border-color: rgb(80, 80, 80);

}</string>
             </property>
             <property name="readOnly">
              <bool>true</bool>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="to_plus_btn">
             <property name="minimumSize">
              <size>
               <width>22</width>
               <height>22</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: white;
	background-color: none;
	border-radius: 5px;
	font-weight: bold;
}

QPushButton:hover {
	border: 1px solid #7f7f7f;
}

QPushButton:pressed {
	background-color: #444444;
}

QPushButton:disabled {
	background-color: rgb(80, 80, 80);
}
</string>
             </property>
             <property name="text">
              <string/>
             </property>
             <property name="icon">
              <iconset resource="../resources/resources.qrc">
               <normaloff>:/icons/icons/add_white_24dp.svg</normaloff>:/icons/icons/add_white_24dp.svg</iconset>
             </property>
            </widget>
           </item>
          </layout>
         </item>
         <item>
          <widget class="QFrame" name="frame">
           <property name="styleSheet">
            <string notr="true">QFrame {
	color: rgb(80,80,80)

}
</string>
           </property>
           <property name="frameShape">
            <enum>QFrame::HLine</enum>
           </property>
           <property name="frameShadow">
            <enum>QFrame::Plain</enum>
           </property>
           <property name="lineWidth">
            <number>1</number>
           </property>
          </widget>
         </item>
         <item>
          <layout class="QHBoxLayout" name="horizontalLayout_4">
           <property name="bottomMargin">
            <number>0</number>
           </property>
           <item>
            <widget class="QPushButton" name="discard_btn">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: white;
	background-color: none;
	border-radius: 5px;
	border: 1px solid #7f7f7f;
	padding: 0px;
}

QPushButton:hover {
	border: 1px solid #f77f00;
}

QPushButton:pressed {
	background-color: #222222;
	color: white;
}

QPushButton:disabled {
	background-color: #555555;
	color: #262626;
	border-color: rgb(80, 80, 80);

}

</string>
             </property>
             <property name="text">
              <string>DISCARD</string>
             </property>
            </widget>
           </item>
           <item>
            <spacer name="horizontalSpacer_3">
             <property name="orientation">
              <enum>Qt::Horizontal</enum>
             </property>
             <property name="sizeHint" stdset="0">
              <size>
               <width>40</width>
               <height>20</height>
              </size>
             </property>
            </spacer>
           </item>
           <item>
            <widget class="QPushButton" name="cancel_btn">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: white;
	background-color: none;
	border-radius: 5px;
	border: 1px solid #7f7f7f;
	padding: 0px;
}

QPushButton:hover {
	border: 1px solid #f77f00;
}

QPushButton:pressed {
	background-color: #222222;
	color: white;
}

QPushButton:disabled {
	background-color: #555555;
	color: #262626;
	border-color: rgb(80, 80, 80);

}

</string>
             </property>
             <property name="text">
              <string>CANCEL</string>
             </property>
            </widget>
           </item>
           <item>
            <widget class="QPushButton" name="confirm_btn">
             <property name="minimumSize">
              <size>
               <width>70</width>
               <height>22</height>
              </size>
             </property>
             <property name="maximumSize">
              <size>
               <width>70</width>
               <height>16777215</height>
              </size>
             </property>
             <property name="styleSheet">
              <string notr="true">QPushButton {
	color: white;
	background-color: #f77f00;
	border-radius: 5px;
	/*border: 1px solid #7f7f7f;*/
	padding: 0px;
}

QPushButton:hover {
	border: 1px solid #f77f00;
}

QPushButton:pressed {	
	background-color: rgb(211, 106, 0);
}

QPushButton:disabled {
	background-color: #555555;
	color: #262626;
	border-color: rgb(80, 80, 80);

}

</string>
             </property>
             <property name="text">
              <string>CONFIRM</string>
             </property>
            </widget>
           </item>
          </layout>
         </item>
        </layout>
       </widget>
      </item>
     </layout>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>QVideoWidget</class>
   <extends>QWidget</extends>
   <header>PyQt5.QtMultimediaWidgets</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources>
  <include location="../resources/resources.qrc"/>
 </resources>
 <connections/>
</ui>
